TelegramBot.py

```diff
@@ -599,7 +599,7 @@ def _cmd_state_system(self, target_id):
         
         statedict = self._get_state_filtered()
         for entity in statedict:
-            if re.match(f"^sensor.({'|'.join(sensorlist)}).*", entity, re.IGNORECASE):
+            if re.match(f"^binary_sensor|^sensor.({'|'.join(sensorlist)}).*", entity, re.IGNORECASE):
```

Comment:
With this change you add the binary_sensor to the /state_system command. Is this indended? I do not think we want every binary sensor listed wih this command. /state_system was introduced to provide a simple command to get the current system health home homeassistant. Thats why a predefined sensorlist is used to filter all available sensors.
```suggestion
            if re.match(f"^sensor.({'|'.join(sensorlist)}).*", entity, re.IGNORECASE):
```

Reply:
I added this so I can see if my door's are closed...

Reply:
Ok, but you somehow misappropriate the /state_system. It was originally introduced to report back entities from https://www.home-assistant.io/integrations/systemmonitor/. 
If you just want to extend the report why don't you use the "extend_system" configuration variable. Just add the complete name of the entities you want to add to the extend_system cofiguration variable seperated by "," (an example is given in the readme). Or is this not sufficient?

Reply:
Ah didn't see that this could be done, okay doorsensors could be done this way. At this time no time to adjust my PR, will look into it as soon as possible...